src/js/jwt/jwt.js

```diff
@@ -237,6 +238,12 @@ exports.getUserInfo = () => {
     });
 };
 
+// Challenge auth and login if the user could be logged in, but in an unauth state
+exports.challengeAuth = () => {
```

Comment:
I hate the name here. That is not a challenge. An auth challenge is whenever a user *must* re-enter their username/pasword.

Doesnt it make way more sense to just export this as login?

Reply:
Same as above. I was just gonna rename the function login

Reply:
Well not login cause thats taken but something like checkAuth